packages/cactus-core/src/test/typescript/unit/plugin-registry.test.ts

```diff
@@ -26,6 +26,10 @@ test("PluginRegistry", (tMain: Test) => {
         throw new Error("This is a mock. Not implemented.");
       },
       getPackageName: () => "@hyperledger/cactus-plugin-keychain-mock",
+
+      onPluginInit: async () => {
+        throw new Error("not sure how this works");
```

Comment:
@TonyRowntree You can make this a no-op as well AFAICT (unless you want to specifically test what happens if the initializer throws - which is not a bad idea by the way)